ArduPlane/px4_mixer.cpp

```diff
@@ -390,6 +390,11 @@ bool Plane::setup_failsafe_mixing(void)
         goto failed;
     }
 
+    if (!(ioctl(px4io_fd, PWM_STATUS, 0) & PX4IO_P_STATUS_FLAGS_MIXER_OK)) {
+        printf("Mixer failed\n");
```

Comment:
Supposed to be a hal.console->printf :) Will fix up now. Spamming is consistent and the appropriate thing.